.github/workflows/main.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     - uses: actions/checkout@v2
     
     - uses: actions/setup-node@v1.4.4
+      with:
+        node-version: '12'
```

Comment:
I'm 👎. This is going to collect dust, especially given that the current version is 14 already.